### DISCUSSION TITLE
Some problems when I upadated Oceananigans on the server

### DISCUSSION THREAD

**Discussion by zwei961120** [CliMA/Oceananigans.jl#2726] [General]
When I tried to update Oceananigans to Version 0.76.8, I met the problem which shows: 

> ERROR: Unsatisfiable requirements detected for package Oceananigans [9e8cae18]:
>  Oceananigans [9e8cae18] log:
>  ├─possible versions are: 0.5.1-0.75.2 or uninstalled
>  └─restricted to versions 0.76.8 by an explicit requirement — no versions left

Is that related to the Version of Julia? My Julia Version on the server is "1.6.6". 
When I tried "pkg> up Oceananigans", it shows

> No Changes to `~/.julia/environments/v1.6/Project.toml`
> No Changes to `~/.julia/environments/v1.6/Manifest.toml`

but my Oceananigans Version is still "0.75.2". How can I update my Oceananigans to the latest Version#v.0.77.2 ?

**Comment by glwagner** [comment]
Try deleting `~/.julia/environments/v1.6/Manifest.toml` and reinstalling?

**Reply by zwei961120** [reply]
Actually I updated Julia version to 1.8.0 and the problem is resolved. Thanks!

**Reply by glwagner** [reply]
Great!
